gopages/go.mod

```diff
@@ -9,8 +9,7 @@ require (
 	github.com/johnstarich/go/pipe v0.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
-	golang.org/x/net v0.0.0-20220421235706-1d1ef9303861
-	golang.org/x/sys v0.0.0-20220328115105-d36c6a25d886 // indirect
-	golang.org/x/tools v0.1.10
+	golang.org/x/mod v0.7.0
+	golang.org/x/net v0.3.0
+	golang.org/x/tools v0.4.0
```

Comment:
I think dropping 1.16 support for `gopages` is okay, especially since it appears to be required to use the latest godoc support. Can you bump up the supported Go version `go 1.16` accordingly, above?

Reply:
Done